web-console/knikubevirt/vm-details/vm-boot-order/vm-boot-order.md

```diff
@@ -62,3 +62,58 @@ The user moves that item with up/down arrow (the screen reader reads the potenti
 
 ![screen-reader - drop item list](img/C1-3-3.jpg)
 The user drops that item on the selected spot
+
+
+## Spec
+
+![Spec-inpage](img/Spec-inpage-01.jpg)
```

Comment:
Should the secondary say `Will boot from disks by order of appearance in YAML file`? small difference but less action sounding.
Nit: Can we make sure the naming convention for the CD-ROMs are the same here to avoid confusion when being implemented. `Drive 1: Container (CD-ROM)`